packages/api/src/errors.js

```diff
@@ -1,7 +1,21 @@
-import { ErrorCode as MagicErrors } from '@magic-sdk/admin'
 import { JSONResponse } from './utils/json-response.js'
 import { DBError } from './utils/db-client.js'
 
+// FIXME: ava gives an error when trying to import ErrorCode from @magic-sdk/admin:
+// SyntaxError: The requested module '@magic-sdk/admin' does not provide an export named 'ErrorCode'
+// Re-declaring here to make progress, but it would be good to figure out how to import it correctly.
+const MagicErrors = {
+  MissingAuthHeader: 'ERROR_MISSING_AUTH_HEADER',
+  TokenExpired: 'ERROR_DIDT_EXPIRED',
+  TokenCannotBeUsedYet: 'ERROR_DIDT_CANNOT_BE_USED_YET',
+  IncorrectSignerAddress: 'ERROR_INCORRECT_SIGNER_ADDR',
+  FailedRecoveryProof: 'ERROR_FAILED_RECOVERING_PROOF',
+  ApiKeyMissing: 'ERROR_SECRET_API_KEY_MISSING',
+  MalformedTokenError: 'ERROR_MALFORMED_TOKEN',
+  ServiceError: 'SERVICE_ERROR',
+  ExpectedBearerString: 'EXPECTED_BEARER_STRING',
+}
+
```

Comment:
any way to avoid this ? did we file an issue ?

Reply:
Oh man, thanks for reminding me - I forgot about this. I'll dig a little deeper and see if I can figure out a solution